cpp/src/parquet/CMakeLists.txt

```diff
@@ -399,7 +400,8 @@ add_parquet_test(arrow-test
                  arrow/arrow_metadata_test.cc
                  arrow/arrow_reader_writer_test.cc
                  arrow/arrow_schema_test.cc
-                 arrow/arrow_statistics_test.cc)
+                 arrow/arrow_statistics_test.cc
+                 arrow/variant_test.cc)
```

Comment:
Why adding a new file? Is `arrow_schema_test.cc` too long?

Reply:
OK, I got it. You are following json_test.cc and uuid_test.cc for canonical extension types.